lib/msprime.c

```diff
@@ -1620,6 +1622,7 @@ msp_recombination_event(msp_t *self)
     self->num_re_events++;
     /* We can't use the GSL integer generator here as the range is too large */
     l = 1 + (int64_t) (gsl_rng_uniform(self->rng) * (double) num_links);
+    printf( "l = %d, num_links = %d\n", (int)l, (int)num_links);
```

Comment:
Yes, this is just a flag for where things are going wrong when I run `./main -v simulate dirac/dev.cfg -r 1`, It looks like recombination hasn't worked on this yet,